### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by Bjox**
Changes:

- New csproj format
- Converted to net6.0
- Updated newtonsoft.json package
- Removed support for snapshots
- New major version (3.0.0)

### CODE REVIEW

**Comment by EL-CTD** [comment]
kis

**Review by EL-CTD — APPROVED** [review]
LGTM

File: Eval.Core/Selection/Parent/TournamentParentSelectionOld.cs [hunk 1]
```diff
@@ -18,7 +18,7 @@
 namespace Eval.Core.Selection.Parent
 {
     [Obsolete("Use new tournament selection instead")]
```
Comment (vikzo) [inline Eval.Core/Selection/Parent/TournamentParentSelectionOld.cs:20]: Since we already have breaking changes, might as well remove obsolete code

**Review by vikzo — APPROVED** [review]

File: Eval.Core/Selection/Parent/TournamentParentSelectionOld.cs (hunk 1 above)
Comment (Bjox) [reply Eval.Core/Selection/Parent/TournamentParentSelectionOld.cs]: Agree

**Review by vikzo — APPROVED** [review]

**Pull request merged**